sa/sa.go

```diff
@@ -402,6 +414,46 @@ func (ssa *SQLStorageAuthority) countCertificatesByName(domain string, earliest,
 	return len(serialMap), nil
 }
 
+// countCertificatesByExactNames returns, for a single domain, the count of
+// certificates issued in the given time range for that domain. In contrast to
+// the countCertificatesByNames function subdomains are NOT considered.
```

Comment:
"In contrast to the countCertificatesByNames function" -> "in contrast to countCertificatesByNames"